### DISCUSSION TITLE
Application shows an error when clicking on "dependabot" in contributors

### DISCUSSION THREAD

**Discussion by DarkWanderer** [community/community#28246] [Mobile]
As stated in title, below error is displayed. I would expect a redirect to documentation website like the web version does

![Screenshot_20220803-214119_GitHub.png](https://user-images.githubusercontent.com/2542609/182695260-8a26cbd7-8c79-4683-a70d-aa908b72592c.png)

**Comment by stevepopovich** [comment]
Hi @DarkWanderer, thanks for filing. I can repro this problem. Will file an issue with the team for prioritization!
